lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -50,6 +50,7 @@ function divideByGCD(fraction) {
     '/', [intermediateNumerator, intermediateDenominator]);
   const status = Node.Status.nodeChanged(
     ChangeTypes.FIND_GCD, fraction, intermediateFraction);
+  status.newNode = Node.Status.resetChangeGroups(status.newNode);
```

Comment:
Looks like this might have been necessary after all -- otherwise in Athena, the whole intermediate node is colored yellow?
![img_1303](https://user-images.githubusercontent.com/1062275/28027373-de3f3c1e-6566-11e7-982f-984d207efcf1.PNG)